src/Content/VersionId.php

```diff
@@ -79,8 +79,12 @@ public static function default(ModelWithContent $model): static
 	/**
 	 * Creates a VersionId instance from a simple string value
 	 */
-	public static function from(string $value): static
+	public static function from(self|string $value): static
```

Comment:
I think, we usually rather use the classname for these cases than `self`

Reply:
static does not work. I've switched to full class names.